### PR TITLE
Enable BraveAdblockCookieListDefault on Nightly/Beta (production)

### PR DESCRIPTION
**Pull request by antonok-edm**
Rebase of https://github.com/brave/brave-variations/pull/188 to the `production` branch, to be merged after verifying the staging PR.

> This feature enables `Easylist-Cookie List - Filter Obtrusive Cookie Notices` from brave://adblock by default, although it can still be overridden by a user who wishes to disable the list.

### CODE REVIEW

File: seed/seed.json
Comment (pes10k) [inline seed/seed.json:72]: Same as on https://github.com/brave/brave-variations/pull/188, this should be 0 right?

**Review by pes10k — DISMISSED** [review]

**Comment by pes10k** [comment]
please wait on merging this for the moment

**Comment by pes10k** [comment]
okie dokie, this is good to merge

**Comment by kjozwiak** [comment]
Verification **PASSED** on `Win 11 x64` using the following builds:

```
Brave | 1.35.75 Chromium: 97.0.4692.71 (Official Build) beta (64-bit)
--- | ---
Revision | adefa7837d02a07a604c1e6eff0b3a09422ab88d-refs/branch-heads/4692@{#1247}
OS | Windows 11 Version 21H2 (Build 22000.376)
```

```
Brave | 1.36.22 Chromium: 97.0.4692.71 (Official Build) nightly (64-bit)
--- | ---
Revision | adefa7837d02a07a604c1e6eff0b3a09422ab88d-refs/branch-heads/4692@{#1247}
OS | Windows 11 Version 21H2 (Build 22000.376)
```

```
Brave | 1.34.80 Chromium: 97.0.4692.71 (Official Build) (64-bit)
-- | --
Revision | adefa7837d02a07a604c1e6eff0b3a09422ab88d-refs/branch-heads/4692@{#1247}
OS | Windows 11 Version 21H2 (Build 22000.376)
```

STR/Cases used:

For each of the following cases, launched the browser using the following:

```
--variations-server-url=https://variations.bravesoftware.com/seed
```

* ensured that `BraveAdblockCookieListDefault:Enabled` via `brave://version` on both Nightly & BETA
* ensured that `Easylist-Cookie List - Filter Obtrusive Cookie Notices` is selected via `brave://adblock`

**`Nightly Example`** | **`Nightly Example`**
------------------------|-------------------------
NightlyEnabled | NightlyAdblockEnabled

**`BETA Example`** | **`BETA  Example`**
------------------------|-------------------------
BETAEnabled | BETAAdblockEnabled

**`Release Example`** | **`Release Example`**
------------------------|-------------------------
ReleaseDisabled | ReleaseAdblockDisabled

**Review by kjozwiak — APPROVED** [review]
Uplift into `production` for `BETA` & `Nightly` at `100%` approved after deliberating with @brave/uplift-approvers. QA has verified the PR as per https://github.com/brave/brave-variations/pull/189#issuecomment-1007647398.

**Pull request merged**
